src/streaming.ts

```
@@ -66,172 +65,150 @@ export class Streaming {
-                console.log(`Client has left`)
-                // Safely close output WebSocket if it exists
-                if (this.output_ws && this.output_ws.readyState === WebSocket.OPEN) {
-                    this.output_ws.close()
+    /**
+     * Setup external output WebSocket (for external services)
+     */
+    private setupExternalOutputWS(): void {
+        try {
+            this.output_ws = new WebSocket(this.outputUrl!)
+
+            this.output_ws.on('open', () => {
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)